src/ripple/app/consensus/LedgerConsensus.cpp

```diff
@@ -1720,10 +1720,10 @@ class LedgerConsensusImp
     */
     void playbackProposals ()
     {
-        for (auto it: getApp().getOPs ().peekStoredProposals ())
+        for (auto const& it: getApp().getOPs ().peekStoredProposals ())
```

Comment:
That's a good observation Scott.  I had to walk through the same logic and came to the same conclusion.